### PR TITLE
Add mathematical functions

### PR DESCRIPTION
**Pull request by Aiee**
Add pi(), e(), radians() to be compatible with openCypher.

fixes https://github.com/vesoft-inc/nebula-graph/issues/717 , fixes https://github.com/vesoft-inc/nebula-graph/issues/718 , fixes https://github.com/vesoft-inc/nebula-graph/issues/719

### CODE REVIEW

**Review by czpmango — APPROVED** [review]
LGTM.

File: src/common/function/FunctionManager.cpp [hunk 1]
```diff
@@ -368,6 +375,17 @@ FunctionManager::FunctionManager() {
...
     }
+    {
+        // return the base of natural logarithm e
+        auto &attr = functions_["e"];
+        attr.minArity_ = 0;
+        attr.maxArity_ = 0;
+        attr.isPure_ = true;
+        attr.body_ = [](const auto &args) -> Value {
+            UNUSED(args);
+            return M_E;
```
Comment (Shylock-Hg) [inline src/common/function/FunctionManager.cpp:386]: kMathE

File: src/common/function/FunctionManager.cpp [hunk 2]
```diff
@@ -449,6 +467,31 @@ FunctionManager::FunctionManager() {
...
     }
+    {
+        // return the mathmatical constant PI
+        auto &attr = functions_["pi"];
+        attr.minArity_ = 0;
+        attr.maxArity_ = 0;
+        attr.isPure_ = true;
+        attr.body_ = [](const auto &args) -> Value {
+            UNUSED(args);
+            return M_PI;
```
Comment (Shylock-Hg) [inline src/common/function/FunctionManager.cpp:478]: ditto.

File: src/common/function/FunctionManager.cpp [hunk 3]
```diff
@@ -368,6 +375,17 @@ FunctionManager::FunctionManager() {
...
         };
     }
+    {
+        // return the base of natural logarithm e
+        auto &attr = functions_["e"];
+        attr.minArity_ = 0;
+        attr.maxArity_ = 0;
+        attr.isPure_ = true;
+        attr.body_ = [](const auto &args) -> Value {
+            UNUSED(args);
```
Comment (Shylock-Hg) [inline src/common/function/FunctionManager.cpp:385]: Check number of arguments.

File: src/common/function/FunctionManager.cpp [hunk 4]
```diff
@@ -449,6 +467,31 @@ FunctionManager::FunctionManager() {
...
         };
     }
+    {
+        // return the mathmatical constant PI
+        auto &attr = functions_["pi"];
+        attr.minArity_ = 0;
+        attr.maxArity_ = 0;
+        attr.isPure_ = true;
+        attr.body_ = [](const auto &args) -> Value {
+            UNUSED(args);
```
Comment (Shylock-Hg) [inline src/common/function/FunctionManager.cpp:477]: ditto.

File: src/common/function/FunctionManager.cpp (hunk 3 above)
Comment (Aiee) [reply src/common/function/FunctionManager.cpp:385]: ```attr.maxArity_ ``` and ```attr.minArity_ ``` should be able to guarantee the number of args is valid

**Review by jievince — APPROVED** [review]

**Review by yixinglu — APPROVED** [review]

**Pull request merged**